build-system/common/check-package-manager.js

```diff
@@ -279,27 +286,37 @@ function runGulpChecks() {
     updatesNeeded.add('gulp');
   }
   if (!firstInstall) {
-    const gulpVersions = getStdout(gulpExecutable + ' --version').trim();
-    const gulpVersion = gulpVersions.match(/Local version[:]? (.*?)$/);
-    if (gulpVersion && gulpVersion.length == 2) {
-      console.log(
-        green('Detected'),
-        cyan('gulp'),
-        green('version'),
-        cyan(gulpVersion[1]) + green('.')
-      );
-    } else {
-      console.log(
-        yellow(
-          'WARNING: ' +
-            'Could not determine the local version of gulp. ' +
-            '(This is normal during install / upgrade.)'
-        )
-      );
-    }
+    printGulpVersion('gulp');
   }
 }
 
+/**
+ * Prints version info for the given gulp command
+ *
+ * @param {string} gulpCmd
+ */
+function printGulpVersion(gulpCmd) {
+  const versionRegex =
+    gulpCmd == 'gulp' ? /Local version[:]? (.*?)$/ : /^CLI version[:]? (.*?)\n/;
+  const gulpVersions = getStdout('gulp --version').trim();
+  const gulpVersion = gulpVersions.match(versionRegex);
+  if (gulpVersion && gulpVersion.length == 2) {
+    console.log(
+      green('Detected'),
+      cyan(gulpCmd),
+      green('version'),
+      cyan(gulpVersion[1]) + green('.')
+    );
+  } else {
+    console.log(
+      yellow(`WARNING: Could not determine the version of ${gulpCmd}.`)
+    );
+  }
+}
+
+/**
+ * Checks if the local version of python is 2.7 or 3
+ */
 function checkPythonVersion() {
   // Python 2.7 is EOL but still supported
   // Python 3.5+ are still supported (TODO: deprecate 3.5 on 2020-09-13)
```

Comment:
Looks like there is a note to deprecate 3.5 on 2020-09-13, worth looking into in a separate PR.